code/modules/antagonists/clockcult/clock_scripture.dm

```diff
@@ -16,6 +16,9 @@ Judgement 5 converts
 	var/chanting = FALSE //If the invocation's words are being spoken
 	var/channel_time = 10 //In deciseconds, how long a ritual takes to chant
 	var/power_cost = 5 //In watts, how much a scripture takes to invoke
+	var/vitality_cost = 0 //how much vitality a scripture requires to be used
+	var/special_vitality_text//if the scripture can use additional vitality to have a unique function, use this; put
+	var/special_vitality_cost
```

Comment:
Should probably be removed, or actually made to have backend support (aka a check for vitality cost / stored vitality on cast & simillar), since right now it does nothing at all.